### PR TITLE
156 improve readme

### PR DESCRIPTION
**Pull request by przemyslawhaptas**

### CODE REVIEW

**Comment by przemyslawhaptas** [comment]
@paulinamaterna You set up the app recently, do you think these changes are enough? There will be another section on automatic emails and possibly something about customizing the app in another PR.

**Comment by przemyslawhaptas** [comment]
[[156] Improve Readme](https://trello.com/c/d8CO4wzF/172-156-improve-readme)

File: README.md [hunk 1]
```diff
@@ -39,3 +49,41 @@ end
...
 
 The configuration for the drivers can be found in the `spec/rails_helper`.
+
+## Registration flow
+  As soon as an applicant sends their submission the following things happen:
+  1. If a submission breaks the default rules (an applicant took part in RailsGirls before or they can't speak English), it is rejected (see "Rejected tab"). Else, it shows in the "Valid" tab.
+  * Admins use "To rate" tab to assess the submission - show a submission, comment on it and give a rate.
+  * If the number of admins who rated a submission is large enough (see "Settings"), the submission is moved from "To rate" tab to "Results" tab.
+  * "Results" hold all rated submission ordered by their average rate. At first, they should have "Invitation" status of "not invited".
+  * You can send an email to the set number of applicants from the top of the list (to set the number see "Settings") - their "Invitation" status should change to "invited".
```
Comment (paulinamaterna) [inline README.md]: You can write something that they will be sent an email with content that is defined in app/views/invitations_mailer

File: README.md [hunk 2]
```diff
@@ -39,3 +49,41 @@ end
...
+  * The invited applicants have some time to confirm the invitations (to set the time see "Settings"). If the invitation is confirmed in the time, the "Invitation" status changes to "confirmed". Else, the status is changed to "expired".
```
Comment (paulinamaterna) [inline README.md]: Invited applicants have some time (to set the time see "Settings") to confirm the invitations. **They do it by clicking on the link in the "invitation email".** If the invitation is confirmed in the time, the "Invitation" status changes to "confirmed".

File: README.md [hunk 3]
```diff
@@ -39,3 +49,41 @@ end
...
+  * All the applicants who confirmed an invitation can be seen in "Participants" tab. It is your participants list, which you can download by clicking a "Download" button.
+
+## Setting automatic emails and reminders
+
+## Adapting the app for other events
+If you go through the process and have to change something that's not mentioned here, please create a PR or drop us a line!
```
Comment (paulinamaterna) [inline README.md]: If you had to change something that is not mentioned here, while going through the process, please...

File: README.md [hunk 4]
```diff
@@ -39,3 +49,41 @@ end
...
+    * title - `app/views/submissions/_form.html.erb`
+    * footer/side pane - `app/views/submissions/new.html.erb`, replace `app/assets/images/rails-girls-krakow-2016.png`
+    * fields
+      * use "QuestionCreator" (admin panel) for managing "how well do you..." questions
+      * to change possible answers to questions to "how well do you..." questions look into
+        * `value` enum in `app/models/answer`
+        * the legend in `app/views/questions/_form.html.erb`
+        * the answer form in `app/views/submissions/_answer_form.html.erb`
+      * removing other fields - the easiest way is to add `visibility="hidden"` and some `value="some value of a correct type"` to the field in `app/views/submissions/_form.html.erb`
+      * adding new fields - well... you probably need to go through the whole flow including view, controller, model, specs, migrations...
```
Comment (paulinamaterna) [inline README.md]: and emails :D

File: README.md [hunk 5]
```diff
@@ -39,3 +49,41 @@ end
...
+  * "Closed" page - `views/submissions/closed.html.erb`
+  * "Invitation confirmed" page - `views/submissions/invitation_confirmed.html.erb`
+  * "Invitation expired" page - `views/submissions/invitation_expired.html.erb`
+  * `404.html`, `422.html`, `500.html`
+  * rejection rules - add or remove rule classes (`app/services/rules`) and reflect the changes in `app/services/submission_rejector`'s `RULES` array
+  * styles - `app/assets/stylesheets`
+  * settings (in the admin panel of the app)
+  * submission form
+    * title - `app/views/submissions/_form.html.erb`
+    * footer/side pane - `app/views/submissions/new.html.erb`, replace `app/assets/images/rails-girls-krakow-2016.png`
```
Comment (paulinamaterna) [inline README.md]: panel

Comment (przemyslawhaptas) [inline README.md]: Oh no, it's on purpose, I'm pretty sure I saw this expression somewhere. :D

**Pull request merged**